e2e/test/Helpers/ConsoleEventListener.cs

```diff
@@ -13,7 +13,7 @@ public sealed class ConsoleEventListener : EventListener
         // OnEventSourceCreated is triggered as soon as the EventListener is registered and an event source is created.
         // So trying to configure this value in the ConsoleEventListener constructor does not work.
         // The OnEventSourceCreated can be triggered sooner than the filter is initialized in the ConsoleEventListener constructor.
-        private static string[] _eventFilters = new string[] { "DotNetty-Default", "Microsoft-Azure-Devices" };
+        private static string[] _eventFilters = new string[] { "DotNetty-Default", "Microsoft-Azure-Devices", "Azure-Core", "Azure-Identity" };
```

Comment:
readonly?

Reply:
For readonly, I did not choose that because we really do not want to change this value in the constructor. It is ideally a const but we cannot declare it as a const due to the new string{}. If you see my comments above, it explains why we should not be changing this value in the constructor. I felt it might be misleading to use readonly.

Reply:
Clarified offline. The comment was to change from 'static' to 'static readonly'. I'll make the change.